packages/core/src/config.ts

```diff
@@ -239,6 +236,71 @@ const composeFormatConfig = (format: Format): RsbuildConfig => {
   }
 };
 
+export const composeModuleImportWarn = (request: string): string => {
+  return `The externalized commonjs request "${request}" will use "module" external type in ESM format. If you want to specify other external type, considering set "output.externals" with the request and type.`;
```

Comment:
how about use a purple or other color for key words like `${request}` `"module"` `"output.externals"`

Reply:
Good idea, I'll give a try.

Reply:
<img width="591" alt="image" src="https://github.com/user-attachments/assets/c7b287ef-f82e-4b59-aa3a-c10533886106">